orders-and-customers/orders/order-items.md

```diff
@@ -123,6 +135,10 @@
 | `meta.display_price.without_tax.amount` | `integer` | The raw total of this product \(excludes tax\) |
 | `meta.display_price.without_tax.currency` | `string` | The currency this display price has been formatted for |
 | `meta.display_price.without_tax.formatted` | `string` | The tax exclusive formatted total based on the currency |
+| `meta.display_price.tax` | `object` | Tax totals |
+| `meta.display_price.tax.amount` | `integer` | The subtotal of the added tax value |
+| `meta.display_price.tax.currency` | `string` | The currency set for the tax |
+| `meta.display_price.tax.formatted` | `string` | The formatted value for the tax subtotal |
```

Comment:
These probably also need updating to reflect `unit`/`value` based on the example above.